pkg/cli/login/login.go

```diff
@@ -47,6 +47,9 @@ var (
 
 		# Log in to the given server through a browser
 		oc login localhost:8443 --web --callback-port 8280
+
+		# Log in to the external OIDC issuer through Auth Code + PKCE by starting a local server listening port 8443
+		oc login localhost:8443 --exec-plugin=oc-oidc --client-id=client-id --extra-scopes=email,profile --callback-port=8443
```

Comment:
Wouldn't this command always fail? You're authenticating to a server `locahost:8443` while also trying to use the `8443` port for the callback, but that port is always busy.

Reply:
Yes, it is correct :). Updated.